### ISSUE TITLE
UnhandledPromiseRejectionWarning

### ISSUE THREAD

**Issue by kraib** [scaleflex/react-cloudimage-responsive#8]
When I import react-cloud-image into my typescript react project. 
`import { CloudimageProvider } from "react-cloudimage-responsive";`
I get this issue.
```(node:39186) UnhandledPromiseRejectionWarning: /Users/kraiba/projects/proggie/eventera.co/frontend/node_modules/react-cloudimage-responsive/dist/index.js:15
import "./polyfills";
       ^^^^^^^^^^^^^

SyntaxError: Unexpected string
    at Module._compile (internal/modules/cjs/loader.js:743:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:810:10)
    at Module.load (internal/modules/cjs/loader.js:666:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:606:12)
    at Function.Module._load (internal/modules/cjs/loader.js:598:3)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at Object.react-cloudimage-responsive (/Users/kraiba/projects/proggie/eventera.co/frontend/.next/dist/bundles/pages/index.js:1550:18)
    at __webpack_require__ (/Users/kraiba/projects/proggie/eventera.co/frontend/.next/dist/bundles/pages/index.js:23:31)
    at Object../src/components/views/HeroBanner/index.tsx (/Users/kraiba/projects/proggie/eventera.co/frontend/.next/dist/bundles/pages/index.js:1151:86)
    at __webpack_require__ (/Users/kraiba/projects/proggie/eventera.co/frontend/.next/dist/bundles/pages/index.js:23:31)
    at Object../src/components/views/EventWithFilters.tsx (/Users/kraiba/projects/proggie/eventera.co/frontend/.next/dist/bundles/pages/index.js:1059:70)
    at __webpack_require__ (/Users/kraiba/projects/proggie/eventera.co/frontend/.next/dist/bundles/pages/index.js:23:31)
    at Object../src/components/containers/Events.tsx (/Users/kraiba/projects/proggie/eventera.co/frontend/.next/dist/bundles/pages/index.js:357:82)
    at __webpack_require__ (/Users/kraiba/projects/proggie/eventera.co/frontend/.next/dist/bundles/pages/index.js:23:31)
    at Object../pages/index.tsx (/Users/kraiba/projects/proggie/eventera.co/frontend/.next/dist/bundles/pages/index.js:85:92)
(node:39186) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:39186) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
{ /Users/kraiba/projects/proggie/eventera.co/frontend/node_modules/react-cloudimage-responsive/dist/index.js:15
import "./polyfills";```
       ^^^^^^^^^^^^^

**Comment by Johbrun** [comment]
Same here !

**Comment by dzmitry-stramavus** [comment]
hey, will fix it right after my vacation

**Comment by Johbrun** [comment]
Hello @dzmitry-stramavus , any news about this ? Thks :)

**Comment by dzmitry-stramavus** [comment]
hey guys, sorry was much work recently

**fixed in version v1.6.5**
